src/main/java/com/knu/buga1chuk/others/ReadFileAndPrintIt.java

```diff
@@ -21,17 +27,20 @@ public static void main(String[] args) {
             try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
                 String string;
                 while ((string = bufferedReader.readLine()) != null) {
-                    System.out.println(string);
+                    //System.out.println(string);
+                    LOG.info(string);
                 }
 
             } catch (IOException e) {
                 String messageFormat = "Error: %s;";
                 String error = e.getMessage();
                 String message = String.format(messageFormat, error);
-                System.out.println(message);
+                //System.out.println(message);
+                LOG.info(message);
```

Comment:
Pay attention here. (it is log in catch)

Reply:
fix